src/posts/datatree/index.md

```diff
@@ -0,0 +1,104 @@
+
+We also had to resolve some really [gnarly design questions](https://github.com/pydata/xarray/pull/9063) to make it work in a way we were happy with.
+
+## How did this happen?
+
+DataTree didn't get implemented overnight - it was a multi-year effort that took place in a number of steps, and there are some lessons to be learned from the story.
+
+In March 2021, the xarray team submitted a [funding proposal](https://zenodo.org/records/5484176) to the [Chan-Zuckerberg Initiative](https://chanzuckerberg.com/eoss/) to develop "TreeDataset", citing bioscience use cases such as [microscopy image pyramids](https://spatialdata.scverse.org/en/latest/design_doc.html). Unfortunately whilst we've been lucky to [receive CZI funding before](https://chanzuckerberg.com/eoss/proposals/xarray-multidimensional-labeled-arrays-and-datasets-in-python/), on this occasion we didn't win money to work on the datatree idea.
+
+In the absence of dedicated funding for datatree, Tom then used some time whilst at the [Climate Data Science Lab](https://ocean-transport.github.io/cds_lab.html) at Columbia University to take a initial stab at the design in August 2021 - writing the first implementation on an overnight Amtrak! This simple prototype was released as a separate package in the [`xarray-contrib/datatree` repository](https://github.com/xarray-contrib/datatree), and steadily gained a small community of intrepid users. It was driven partly by the use case of [climate model intercomparison datasets](https://medium.com/pangeo/easy-ipcc-part-1-multi-model-datatree-469b87cf9114).
+
+A separate repository was chosen for speed of iteration, and to be able to more easily [make changes](https://github.com/xarray-contrib/datatree/blob/7ba05880c37f2371b5174f6e8dcfae31248fe19f/README.md#development-roadmap) without worrying as much about [backwards compatibility](https://github.com/pydata/xarray/issues/9854) as code in xarray's main repo does. However the separate repo meant that the prototype `datatree` library was not fully integrated with xarray's main codebase, limiting possible features and requiring fragile dependencies on private xarray internals.
+
+The prototype then sat there for 2 years, until the NASA ESDIS team approached the xarray core team in August 2023. ESDIS devs wanted the ability to work with entire hierarchical files, and had experimented with the prototype version of datatree, but they wanted datatree functionality to be migrated upstream into xarray's main repository so there would be more guarantees of long-term API stability and support.
+
+Amazingly the NASA team were able to offer engineer time, so starting in late 2023 Owen, Matt, and Eni (NASA) worked on migrating the prototype datatree into xarray upstream, with regular supervision from Tom, Justus, and Stephan (existing xarray team).
```

Comment:
I just checked and folks at my org would like my full affiliation. I expect @eni-awowale and @owenlittlejohns will also need full affiliations. I see that it is unruly and long.
```
Matt (NASA National Snow and Ice Data Center Distributed Active Archive Center)
```

Reply:
I guess just to throw out more detailed affiliations for Eni and I:

* I am on the NASA "EOSDIS Evolution and Development 3" (EED-3) contract.
* Eni is part of the Goddard Earth Sciences Data and Information Services Center (GES DISC) - but I'm not 100% sure how GES DISC prefer that being represented.

Reply:
Added in https://github.com/xarray-contrib/xarray.dev/pull/736/commits/9dc4aafacf9d331775fd8ac8cbc08feeec996580